contracts/script/EigenDADeployer.s.sol

```diff
@@ -409,5 +411,21 @@ contract EigenDADeployer is DeployOpenEigenLayer {
             address(eigenDARelayRegistryImplementation),
             abi.encodeWithSelector(EigenDARelayRegistry.initialize.selector, addressConfig.eigenDACommunityMultisig)
         );
+
+        // Deploy EigenDAEjectionManager
+        // Using the first deployed strategy token as deposit token
+        address depositToken = address(deployedStrategyArray[0].underlyingToken());
+        uint256 depositBaseFeeMultiplier = 100; // 100x base fee multiplier
+        uint256 estimatedGasUsedWithoutSig = 100000; // 100k gas estimate
+        uint256 estimatedGasUsedWithSig = 200000; // 200k gas estimate with signature verification
+
+        eigenDAEjectionManager = new EigenDAEjectionManager(
+            depositToken,
+            depositBaseFeeMultiplier,
+            address(eigenDADirectory),
+            estimatedGasUsedWithoutSig,
+            estimatedGasUsedWithSig
+        );
+        eigenDADirectory.addAddress(AddressDirectoryConstants.EIGEN_DA_EJECTION_MANAGER_NAME, address(eigenDAEjectionManager));
```

Comment:
maybe you also want to call eigenDAEjectionManager.setCooldown(x) and setDelay(x)